flags/14.0/vtgate.txt

```diff
@@ -1,48 +1,48 @@
 Usage of vtgate:
-      --allowed_tablet_types value                                       Specifies the tablet types this vtgate is allowed to route queries to
+      --allowed_tablet_types []topodatapb.TabletType                     Specifies the tablet types this vtgate is allowed to route queries to.
```

Comment:
Is there a way we can make this string? and it's supposed to be comma-separated, right?

Reply:
@ajm188 ☝️

Reply:
Not sure what your question is exactly, tbh. This is a list of tablet types, which can be both comma separated (`"replica,primary"`) and by specifying the flag multiple times

Reply:
if the question is about the type (`value` => `[]topodatapb.TabletType`) that's just coming from [here](https://github.com/vitessio/vitess/blob/main/go/vt/topo/topoproto/flag.go#L42), but you can see in the actual [`Set`](https://github.com/vitessio/vitess/blob/main/go/vt/topo/topoproto/flag.go#L36) that the behavior is as i've described

Reply:
to clarify my question: when the users are seeing the help text, they should not need to know what `topodatapb.TabletType` is. They just need to know to provide a CSV. so it will be good if we can somehow make this print
```
--allowed_tablet_types string      Specifies the tablet types this vtgate is allowed to route queries to. Should be provided as a comma-separated set of tablet types
```
or something like that.

Reply:
yeah, so this comes from the `Type()` method on a `pflag.Value` so you can make it say whatever you want (you shouldn't do that though - e.g. `func (f *myFlag) Type() string { "crashes-your-program-and-steals-your-data" }` :P

Reply:
done